lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -110,6 +110,11 @@ def self.create_automation_request(options, userid = "admin", auto_approve = fal
       MiqAeServiceModelBase.wrap_results(AutomationRequest.create_request(options, user, auto_approve))
     end
 
+    def self.create_service_provision_request(svc_template, options = nil)
+      result = ar_object(svc_template).provision_request(@workspace.ae_user, options)
+      MiqAeServiceModelBase.wrap_results(result)
+    end
```

Comment:
This method has to change a bit more now that it is running under a different class.  Since we do not have access to `@workspace` we can get the user from the `User.current_user` set but the calling method `execute` [here](https://github.com/ManageIQ/manageiq-automation_engine/blob/2c4b6e9a4813ba142d528b403acdbd1f7916d193/lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service.rb#L168).

```ruby
def self.create_service_provision_request(svc_template, options = nil)
  result = svc_template.object_send(:provision_request, User.current_user, options)
  MiqAeServiceModelBase.wrap_results(result)
end
```